docs/user/gcp/install_upi.md

```diff
@@ -0,0 +1,593 @@
+
+Then, the bootstrap node can be removed.
+
+``` sh
+gcloud compute target-pools remove-instances ${INFRA_ID}-api-target-pool --instances=${INFRA_ID}-bootstrap
+gcloud compute target-pools remove-instances ${INFRA_ID}-ign-target-pool --instances=${INFRA_ID}-bootstrap
+```
+## Destroy bootstrap resources
+
+At this point, you should delete the bootstrap resources.
+
+```sh
+gsutil rm gs://${INFRA_ID}-bootstrap-ignition/bootstrap.ign
+gsutil rb gs://${INFRA_ID}-bootstrap-ignition
+gcloud deployment-manager deployments delete ${INFRA_ID}-bootstrap
+```
+
+## Launch additional compute nodes
+
+You may create compute nodes by launching individual instances discretely
+or by automated processes outside the cluster (e.g. Auto Scaling Groups). You
+can also take advantage of the built in cluster scaling mechanisms and the
+machine API in OpenShift, as mentioned [above](#create-ignition-configs). In
+this example, we'll manually launch one instance via the Deployment Manager
+template. Additional instances can be launched by including additional
+resources of type 06_worker.py in the file.
+
+Copy [`06_worker.py`](../../../upi/gcp/06_worker.py) locally.
+
+Export variables needed by the resource definition.
+
+```sh
+export COMPUTE_SUBNET=`gcloud compute networks subnets describe ${INFRA_ID}-worker-subnet --format json | jq -r .selfLink`
+export WORKER_SERVICE_ACCOUNT_EMAIL=`gcloud iam service-accounts list | grep "^${INFRA_ID}-worker-node " | awk '{print $2}'`
+export WORKER_IGNITION=`cat worker.ign`
+```
+
+Create a resource definition file: `06_worker.yaml`
+```console
+$ cat <<EOF >06_worker.yaml
+imports:
+- path: 06_worker.py
+
+resources:
+- name: 'w-a-0'
+  type: 06_worker.py
+  properties:
+    infra_id: '${INFRA_ID}'
+    region: '${REGION}'
+
+    compute_subnet: '${COMPUTE_SUBNET}'
+    image: '${CLUSTER_IMAGE}'
+    machine_type: 'n1-standard-4'
+    root_volume_size: '128'
+    service_account_email: '${WORKER_SERVICE_ACCOUNT_EMAIL}'
+    zone: '${REGION}-a'
+
+    ignition: '${WORKER_IGNITION}'
+EOF
+```
+- `name`: the name of the compute node (for example w-a-0)
+- `infra_id`: the infra_id from above (for example openshift-xxxx)
+- `region`: the region to deploy the cluster into (for example us-east1)
+- `compute_subnet`: the URI to the compute subnet
+- `image`: the URI to the RHCOS image
+- `machine_type`: The machine type of the instance (for example n1-standard-4)
+- `service_account_email`: the email address for the worker service account created above
+- `zone`: the zone for the worker node (for example us-east1-b)
+- `ignition`: the contents of the worker.ign file
+
+Create the deployment using gcloud.
+
+```sh
+gcloud deployment-manager deployments create ${INFRA_ID}-worker --config 06_worker.yaml
+```
+
+#### Approving the CSR requests for nodes
+
+The CSR requests for client and server certificates for nodes joining the cluster will need to be approved by the administrator.
+You can view them with:
+
+```console
+$ oc get csr
```

Comment:
You should mention before this that they should set their KUBECONFIG var so `oc` looks at your new cluster

Reply:
Done.